app/controllers/api/service_dialogs_controller.rb

```diff
@@ -26,8 +26,8 @@ def create_resource(_type, _id, data)
     def edit_resource(type, id, data)
       service_dialog = resource_search(id, type, Dialog)
       begin
-        service_dialog.update!(data.except('content'))
-        service_dialog.update_tabs(data['content']['dialog_tabs']) if data['content']
+        service_dialog.update!(data.except('dialog_tabs', 'content'))
+        service_dialog.update_tabs(data['dialog_tabs'] || data['content']['dialog_tabs']) if data['dialog_tabs'] || data['content']
```

Comment:
if both are specified, we're looking at dialog_tags instead of the current content, is that the expectation ?

also, we need specs updates to test both cases.

/cc @jntullo